cmd/es-rollover/app/lookback/action.go

```diff
@@ -54,16 +57,19 @@ func (a *Action) lookback(indexSet app.IndexOption) error {
 	finalIndices := filter.ByDate(excludedWriteIndex, getTimeReference(timeNow(), a.Unit, a.UnitCount))
 
 	if len(finalIndices) == 0 {
-		return fmt.Errorf("no indices to remove from alias %s", readAliasName)
+		a.Logger.Info(fmt.Sprintf("no indices to remove from alias %s", readAliasName))
```

Comment:
Please pass parameters as zap parameters only. e.g. `logger.Info("Message", zap.String("param-name", "foo")`

Also prefer to start the log message with a capital letter.

Reply:
Thanks @pavolloffay, updated